<launch>

  <arg name="imu_filter"  default="true"/>
  <arg name="imu_port"  default="/dev/transducer"/>

  <rosparam param="imu_port" subst_value="True">$(arg imu_port)</rosparam>

  <node pkg="tf"
        type="static_transform_publisher"
        name="base_link_to_imu_broadcaster"
        args="0 0 0 0 0 0 base_link imu 100" />

  <node pkg="imu_filter_madgwick"
        type="imu_filter_node"
        name="imu_filter_node">
    <remap from="/imu/data_raw" to="/imu_data_raw" />
    <remap from="/imu/mag"      to="/imu_data_mag" />
    <remap from="/imu/data"     to="/imu_data"/>
  </node>

  <node pkg="tuw_sensor_transducer"
        type="tuw_sensor_transducer_node"
        name="tuw_sensor_transducer_node"
        output="screen"/>

</launch>